helm/netobserv/Chart.yaml

```
@@ -0,0 +1,24 @@
+apiVersion: v2
+name: Netobserv
+description: A Helm chart to install Network Observability Operator on k8s
+
+# A chart can be either an 'application' or a 'library' chart.
+#
+# Application charts are a collection of templates that can be packaged into versioned archives
+# to be deployed.
+#
+# Library charts provide useful utilities or functions for the chart developer. They're included as
+# a dependency of application charts to inject those utilities and functions into the rendering
+# pipeline. Library charts do not define any templates and therefore cannot be deployed.
+type: application
+
+# This is the chart version. This version number should be incremented each time you make changes
+# to the chart and its templates, including the app version.
+# Versions are expected to follow Semantic Versioning (https://semver.org/)
+version: 1.0.0
```

I think we should start with a pre-1.0.0 until we feel it's ready to be published on ArtifactHub

```suggestion
version: 0.0.1
```

---

got it..